### ISSUE TITLE
remove api key from repo

### ISSUE THREAD

**Issue by jhsu** [themcstaplez/lol-api-react#1]
you might want to remove your key and revoke access (since it's public now) and move it to a config file

https://github.com/themcstaplez/lol-api-react/blob/b88a16895f5d97d2b49ba933b9ed434158854dd6/js/main.js

**Comment by juiceByStaplez** [comment]
Would it seem logical to have a config.js with this information and then use something like 'require('./config') in the main.js while ignoring the config.js from this repo?

Thanks for bringing attention to this.

**Comment by jhsu** [comment]
yeah, and add an example config for reference `config.example.js` with dummy data.

Then, you could require it in `main.js` and pass it as a prop to `<LeagueSearch apiKey={yourApiKey} />`
